cocos/3d/assets/mesh.ts

```diff
@@ -1012,10 +1016,6 @@ export class Mesh extends Asset {
             data: globalEmptyMeshBuffer,
         });
     }
-
-    public validate () {
-        return this.renderingSubMeshes.length > 0 && this.data.byteLength > 0;
-    }
```

Comment:
Why remove `validate` function ? And which problem this PR wants to solve

Reply:
Can not validate mesh correctly, so move initialize function to `onloaded` and use `try-catch` to validate it

Reply:
This validation is not accurate and groundless, it broke empty meshes.

Reply:
If using JS try-catch to validate, how to do it in C++?